### ISSUE TITLE
Not working in IOS9

### ISSUE THREAD

**Issue by sandeeppster** [montagejs/beachplanetblog#11]
Hello,

This example is not working in IOS9.
It was earlier working in IOS8. Can it be looked into/

Regards,
Sandeep Patil

**Comment by marchant** [comment]
yes, thanks for letting us know, there are some changes between iOS8->9 that are causing this, we're looking into it

**Comment by sandeeppster** [comment]
Hello Marchant, I hope this will work in IOS 9 soon.
